src/services/i18n.service/index.ts

```diff
@@ -5,10 +5,24 @@ import { Locale } from './enums';
 import locales from './locales';
 
 class I18nService {
-  currentLocale: string = Locale.English;
+  defaultLocale: string = Locale.English;
 
   get(key: string, values = {}): string {
-    const messageTemplate = _.get(locales[this.currentLocale], key) || key;
+
+    let lang = this.defaultLocale;
+
+    switch(navigator.language){
```

Comment:
Codacy has a fix for the issue: [Replace `(navigator.language)` with `·(navigator.language)·`](https://app.codacy.com/gh/internxt/drive-web/pullRequest?prid=9469503)
```suggestion
    switch (navigator.language) {
```